src/components/Header/TopNav/index.js

```diff
@@ -34,12 +35,56 @@ const TopNavList = styled.ul`
   }
 `
 
-const TopNav = ({ links, themeVariation, path }) => (
+const getSlugs = (arr = []) => arr.map(({ slug }) => slug).filter(i => i)
```

Comment:
`const getSlugs = (arr = []) => arr.filter(({ slug }) => slug)`

No need to map before filter ;)

Reply:
@fabiommmoreira I removed to just be filter but I actually wanted to get the slugs then filter the array, not filter the array based on whether the object at current index had valid slugs.